industry_benchmarks/input_structures/prepared_structures/fragments/mcl1/PREPARATION_DEATILS.md

```diff
@@ -0,0 +1,9 @@
+
+protein:
+ - used maestro protein preparation to cap C-term histidine
```

Comment:
```suggestion
 - used maestro protein preparation to cap C-term histidine
 - the N-term was kept charged since this is the actual beginning of the construct 
```